automatic/pdfcreator/tools/chocolateyInstall.ps1

```diff
@@ -36,22 +36,30 @@ try {
   Write-Host "Print Spooler service state: $($spoolerService.StartMode) / $($spoolerService.Status)"
   if ($spoolerService.StartMode -ne 'Auto' -or $spoolerService.Status -ne 'Running') {
     Set-Service $serviceName -StartupType Automatic -Status Running
-    Write-Host "Print Spooler service new state: Auto / Running"
+    Write-Host 'Print Spooler service new state: Auto / Running'
   }
 }
 catch {
   Write-Warning "Unexpected error while checking Print Spooler service: $($_.Exception.Message)"
 }
 
-# silent install requires AutoHotKey
-$ahkFile = Join-Path $toolsPath 'chocolateyInstall.ahk'
-$ahkEXE = Get-ChildItem "$env:ChocolateyInstall\lib\autohotkey.portable" -Recurse -filter autohotkey.exe
-$ahkProc = Start-Process -FilePath $ahkEXE.FullName -ArgumentList "$ahkFile" -PassThru
-Write-Debug "AutoHotKey start time:`t$($ahkProc.StartTime.ToShortTimeString())"
-Write-Debug "Process ID:`t$($ahkProc.Id)"
+# silent install requires different methods for domain-joined systems
```

Comment:
Too much details in main installer. Please make it use `helper.ps1` with adequately named functions. See [1.1.7](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#117-clean-code)

Reply:
All of that AHK "stuff" is in the current package, I just made its use dependent on domain-joined-status.  It's interesting that including it was acceptable the last time.

I'm not sure how I would separate that into separate functions.  Are there other packages that use AHK that do it "properly"?  

If using AHK is too detailed for the main installer and a common "issue", someone should make a Chocolatey extension package to handle in a standardized way.

Reply:
I am not talking about AHK,  but about `chocolateyInstall.ps1`. Yeah, there are number of packages, just hit Find file here and look for `helpers.ps1` files.

Reply:
I'm sorry, but I not sure I understand.  What detail needs to be moved to a function?  The AHK process creation is the same as before.  I added an "if part of domain" and a couple messages for the user.

Reply:
I added an intermediate step to make it more readable.